store/src/java/com/zimbra/cs/service/util/VolumeConfigUtil.java

```diff
@@ -299,7 +302,27 @@ public static void parseModifyVolumeRequest(ModifyVolumeRequest req) throws Serv
         // store type == 2, allow modification of only volume name
         else if (vol.getStoreType().equals(Volume.StoreType.EXTERNAL)) {
             if (volInfo.getName() != null) {
+                String storageType = "";
+                String globalBucketConfigId = "";
+                ExternalVolumeInfoHandler extVolInfoHandler = new ExternalVolumeInfoHandler(Provisioning.getInstance());
+                try {
+                    JSONObject properties = extVolInfoHandler.readServerProperties(req.getId());
+                    storageType = properties.getString(AdminConstants.A_VOLUME_STORAGE_TYPE);
+                    globalBucketConfigId = properties.getString(AdminConstants.A_VOLUME_GLB_BUCKET_CONFIG_ID);
+                } catch (JSONException e) {
+                    throw ServiceException.FAILURE("Error while reading json data for external volume: " + req.getId(), e);
+                }
+                // storageType should not be null
+                if (StringUtil.isNullOrEmpty(storageType)) {
+                    throw VolumeServiceException.INVALID_REQUEST("StorageType Empty for external volume " +req.getId(), VolumeServiceException.INVALID_REQUEST);
```

Comment:
add space after + 
change `StorageType Empty for external volume` to `Storage type empty for external volume`

Reply:
Done, fixed